Fitness/Controllers/HomeController.swift

```diff
@@ -121,23 +126,26 @@ class HomeController: UIViewController {
 extension HomeController: UICollectionViewDataSource {
 
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        if collectionView != mainCollectionView {
+        if collectionView.accessibilityIdentifier == Identifiers.todaysClassesCell {
```

Comment:
Does this work? In similar changes I made i had `if collectionView == todaysClassesCollectionView`, although I had to actually set the `todaysClassesCollectionView` since it currently isn't

Reply:
As far as I can tell it works, though I suppose I could have also created references in the `HomeController` to each of the different sections' individual collection views as well. Should I change it?